Unix/installbuilder/conf/omiserver.conf

```diff
@@ -58,6 +60,3 @@ keyfile=/etc/opt/omi/ssl/omikey.pem
 configfile=/etc/opt/omi/conf/omiserver.conf
 #NoSSLv2=true
 #NoSSLv3=false
-
-## non-root feature parameters
-#nonroot=true
```

Comment:
Hip hip hooray. I never liked this here. Even if it worked (and I don't think it does anymore), we don't suggest it and wouldn't support it anymore.